base_partner_merge/wizard/base_partner_merge.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Camptocamp SA
+# Copyright 2017 Jarsa Sistemas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.addons.crm.wizard.base_partner_merge import *  # noqa
```

Comment:
seems we import only 2 things: `MergePartnerLine` and `MergePartnerAutomatic` -> remove the wildcard